monitor/census.go

```diff
@@ -176,6 +184,13 @@ func InitCensus(nodeType, nodeID, version string) {
 	if err != nil {
 		glog.Fatal("Error creating context", err)
 	}
+	census.mHTTPClientTimeout1 = stats.Int64("http_client_timeout_1", "Number of times HTTP connection was dropped before transcoding complete", "tot")
+	census.mHTTPClientTimeout2 = stats.Int64("http_client_timeout_2", "Number of times HTTP connection was dropped before transcoded segments was sent back to client", "tot")
+	census.mRealtime3x = stats.Int64("segment_transcoded_realtime_3x", "Number of segment transcoded 3x faster than realtime", "tot")
```

Comment:
Should these names also have the "http_client_" prefix to clarify that the metrics are measured from the POV of the HTTP client and *not* the POV of the broadcaster since we have other metrics such as `transcode_overall_latency_seconds` that are measured from the POV of the broadcaster?

Reply:
fixed in 1d573763b9e6c362b8e1473f7a675194395fd9fe